lib/libmuser.c

```diff
@@ -196,7 +197,8 @@ irqs_set_data_bool(lm_ctx_t *lm_ctx, struct vfio_irq_set *irq_set, void *data)
 
     assert(data != NULL);
 
-    for (i = irq_set->start, d8 = data; i < irq_set->start + irq_set->count;
+    for (i = (int)irq_set->start, d8 = data;
```

Comment:
Same 'i' can be declared as unsigned int.

Reply:
fixed